src/server/helpers/source-api-request.ts

```diff
@@ -106,6 +95,8 @@ export async function requestData<T>(
     );
   }
 
+  const debugResponseData = getDebugResponseData(requestConfig);
```

Comment:
Dit lijkt niet te gaan werken, de functie die je retourneert is altijd nieuw en is dus nooit als transformer toegevoegd. Bij elk request naar een api voeg je opnieuw deze transformer toe.